<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBookWindow</class>
 <widget class="QDialog" name="addBookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#2d2d2d;&quot;&gt;ADICIONAR UM LIVRO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBookButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(
    x1:0, y1:0, x2:1, y2:0,
    stop:0 #8B0000, stop:1 #ED1C24
);
color: white;
border: none;
padding: 12px 20px;
font-size: 15px;
font-weight: bold;
border-radius: 10px;
text-align: center;

</string>
   </property>
   <property name="text">
    <string>ADICIONAR</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>57</y>
     <width>391</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#2d2d2d;&quot;&gt;TEMA:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="namelineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;
border: 1px solid #CCCCCC;
padding: 8px;
border-radius: 8px;
color: #2D2D2D;
outline: none; /* Remove a borda de foco padrão do SO */
font: 700 10pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#2d2d2d;&quot;&gt;AUTOR:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="authorlineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;
border: 1px solid #CCCCCC;
padding: 8px;
border-radius: 8px;
color: #2D2D2D;
outline: none; /* Remove a borda de foco padrão do SO */
font: 700 10pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#2d2d2d;&quot;&gt;NOME:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="topiclineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;
border: 1px solid #CCCCCC;
padding: 8px;
border-radius: 8px;
color: #2D2D2D;
outline: none; /* Remove a borda de foco padrão do SO */
font: 700 10pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#2d2d2d;&quot;&gt;ANO LANÇAMENTO:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yearlineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: white;
border: 1px solid #CCCCCC;
padding: 8px;
border-radius: 8px;
color: #2D2D2D;
outline: none; /* Remove a borda de foco padrão do SO */
font: 700 10pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
